test/token/ERC20/ERC20.test.js

```diff
@@ -287,6 +287,46 @@ contract('ERC20', function (accounts) {
     });
   });
 
+  describe('_update', function () {
+    const amount = 1;
+    
+    it('from is the zero address', async function () {
+      const totalSupply = await this.token.totalSupply();
+      const newSupply = totalSupply.add(new BN(amount));
+      await this.token.update(ZERO_ADDRESS, recipient, amount);
+
+      expect(await this.token.totalSupply()).to.be.bignumber.equal(newSupply);
+      expect(await this.token.balanceOf(recipient)).to.be.bignumber.equal(amount);
+    });
+
+    it('to is the zero address', async function () {
```

Comment:
Let's add an assertion for the transfer event in this test.

Reply:
Fixed in last commit.